terraform/root_production.tf

```diff
@@ -0,0 +1,125 @@
+  ec2_instance_id = module.jenkins_ec2_prod.instance_id
+  name            = "tdr-jenkins-backup-prod-window"
+  schedule        = "cron(0 0 18 ? * MON-FRI *)"
+  common_tags     = local.common_tags
+}
+
+module "jenkins_maintenance_window_event_prod" {
+  source                  = "./tdr-terraform-modules/cloudwatch_events"
+  event_pattern           = "jenkins_maintenance_event_window"
+  lambda_event_target_arn = list(data.aws_lambda_function.notifications_function.arn)
+  rule_name               = "jenkins-backup-maintenance-window-prod"
+  rule_description        = "Capture failed runs of the jenkins backup"
+  event_variables         = { window_id = module.jenkins_backup_maintenance_window_prod.window_id }
+}
+
+module "jenkins_integration_fargate_policy_prod" {
+  source        = "./tdr-terraform-modules/iam_policy"
+  name          = "TDRJenkinsFargateProdPolicy${title(local.environment)}"
+  policy_string = templatefile("./tdr-terraform-modules/iam_policy/templates/jenkins_fargate_prod.json.tpl", { account_id = data.aws_caller_identity.current.account_id })
+}
+
+module "jenkins_prod_fargate_role" {
+  source             = "./tdr-terraform-modules/iam_role"
+  common_tags        = local.common_tags
+  assume_role_policy = templatefile("./tdr-terraform-modules/iam_policy/templates/assume_role_policy.json.tpl", { role_arn = module.jenkins_prod_ecs_task_role.role.arn })
+  name               = "TDRJenkinsFargateRoleProd${title(local.environment)}"
+  policy_attachments = { fargate_policy = module.jenkins_integration_fargate_policy_prod.policy_arn, ssm_core = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore" }
+}
+
+module "jenkins_certificate_prod" {
+  source      = "./tdr-terraform-modules/certificatemanager"
+  project     = var.project
+  function    = "jenkins-prod"
+  dns_zone    = var.dns_zone
+  domain_name = "jenkins-prod.tdr-management.nationalarchives.gov.uk"
+  common_tags = local.common_tags
+}
+
+module "jenkins_alb_prod" {
+  source                           = "./tdr-terraform-modules/alb"
+  project                          = var.project
+  function                         = "jenkins-prod"
+  environment                      = local.environment
+  alb_log_bucket                   = module.jenkins_logs_s3.s3_bucket_id
+  alb_security_group_id            = module.jenkins_alb_security_group.security_group_id
+  certificate_arn                  = module.jenkins_certificate_prod.certificate_arn
+  health_check_unhealthy_threshold = 5
+  domain_name                      = "jenkins.tdr-management.nationalarchives.gov.uk"
+  public_subnets                   = module.jenkins_vpc.public_subnets
+  target_id                        = module.jenkins_ec2_prod.instance_id
+  vpc_id                           = module.jenkins_vpc.vpc_id
+  common_tags                      = local.common_tags
+}
+
+module "jenkins_s3_backup_prod_policy" {
```

Comment:
Should this resource be renamed: `jenkins_s3_backup_policy_prod` to make it consistent?